src/main/java/org/example/model/Customer.java

```diff
@@ -0,0 +1,56 @@
+package org.example.model;
+
+import java.util.ArrayList;
+import java.util.List;
+import javax.persistence.*;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+@Entity
+@NoArgsConstructor
+@Getter
+public class Customer {
+  @Id
+  @GeneratedValue(strategy = GenerationType.IDENTITY)
+  private Long id;
+
+  private String name;
+  private String email;
+  private String address;
+  private String phone;
+
+  @OneToMany(mappedBy = "customer", cascade = CascadeType.REMOVE, orphanRemoval = true)
+  private List<Order> orders = new ArrayList<>();
```

Comment:
Move the initialization of this attribute to the constructor.

`this.orders = new ArrayList<>();`

Reply:
Done.